public/index.html

```diff
@@ -10,6 +10,11 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="logo192.png" />
+    <link
+      rel="stylesheet"
+      href="https://fontlibrary.org/face/segment7"
+      type="text/css"
+    />
```

Comment:
So you've aded a font library but the component still looks like it's from the 90's

Reply:
I put that in my Backlog, thanks